web/src/components/storage/LvmPage.tsx

```diff
@@ -57,14 +57,18 @@ import { _ } from "~/i18n";
  * that are already there.
```

Comment:
The comment is outdated, right?

Reply:
No. Is not. That function (and the comment) is still needed until we allow to add all kind of RAIDs (and not only the candidates) in the UI. I will write that to the comment.

Reply:
Comment expanded